kytos/core/logs.py

```diff
@@ -3,34 +3,32 @@
 import re
 from configparser import RawConfigParser
 from logging import Formatter, StreamHandler, config, getLogger
+from pathlib import Path
 
 __all__ = ('LogManager', 'NAppLog')
 
 
 class LogManager:
     """Manage handlers for all loggers."""
 
-    configuration = None
+    _CONFIG = RawConfigParser()
+    _DEFAULT_FMT = 'formatter_console'
 
     @classmethod
-    def load_logging_file(cls, logging_file):
-        """Loadding logs configuration from a file.
+    def load_logging_file(cls, config_file):
+        """Load log configuration file.
 
         Args:
-           logging_file(str): Address of logging configuration file.
+           config_file (str, Path): Configuration file path.
         """
-        cls.configuration = RawConfigParser()
-        cls.configuration.read(logging_file)
-
-        try:
-            config.fileConfig(logging_file)
-        except FileNotFoundError:
-            cls.configuration.set('handler_syslog', 'args', '[]')
```

Comment:
I guess you forgot to verify the handler_syslog. This line change the 'handler_syslog' if the file /dev/syslog don't exists.